projects/ion/src/lib/input/input.component.html

```diff
@@ -75,3 +75,7 @@
     <ion-icon type="close-solid"></ion-icon>
   </button>
 </div>
+
+<div class="errorMsg">
```

Comment:
```suggestion
<div class="errorMsg" *ngIf="invalid">
```
Provavelmente seria melhor aqui para não adicionar sempre essa div com essa classe.